main.py

```diff
@@ -565,3 +548,29 @@ def handle_stats_reply(self, msg, switch):
             event = KytosEvent('amlight/flow_stats.flows_updated')
             event.content['switch'] = switch.dpid
             self.controller.buffers.app.put(event)
+
+    @listen_to('kytos/of_core.flow_stats.received')
+    def on_stats_received(self, event):
+        """Capture flow stats messages for OpenFlow 1.3."""
+        self.handle_stats_received(event)
+
+    def handle_stats_received(self, event):
+        """Handle flow stats messages for OpenFlow 1.3."""
+        switch = event.content['switch']
+        if 'replies_flows' in event.content:
+            replies_flows = event.content['replies_flows']
+            self.handle_stats_reply_received(switch, replies_flows)
+
+    @classmethod
+    def handle_stats_reply_received(cls, switch, replies_flows):
+        """Iterate on the replies and set the generic flows"""
+        flows = []
+        for flow in replies_flows:
+            flows.append(GenericFlow.from_flow_stats(flow, switch.ofp_version))
+        if 'generic_flows' not in switch:
+            switch.generic_flows = []
```

Comment:
@gretelliz this code that you've refactored isn't equivalent, you're initializing a list, and keep extending it, it'll keep copying duplicated flows. `generic_flows` is always supposed to be reassigned. As mentioned in the prior comments, you can replace most of this code with a single list compression, something like:

```switch.generic_flows = sorted([GenericFlow.from_flow_stats(flow, switch.ofp_version) for flow in replies_flows], key=lambda f: (f.priority, f.duration_sec), reverse=True)```

Also, make sure to cover with unit test this `handle_stats_reply_received`, it's not being covered.

Reply:
This PR as it is will keep crashing, you'll likely need to implement a new method on `GenericFlow` to map it from [`Flow` aka `Flow04`](https://github.com/kytos-ng/of_core/blob/master/v0x04/flow.py#L328), these are the instances that are being passed on `replies_flows`. Make sure to test and explore it locally in addition to unit testing it. When you need me to review this PR again, you can either click on the request review button or reply on tag me in the comments, thanks.

Reply:
Also, make sure to update the CHANGELOG.rst file, you can add a line in the `[UNRELEASED] - Under development` section under the `Changed` category explaining the change that's being shipped here.